src/Tribe/Views/V2/Hooks.php

```diff
@@ -117,6 +118,27 @@ protected function add_filters() {
 		add_filter( 'tribe_events_views_v2_view_data', [ View_Utils::class, 'clean_data' ] );
 	}
 
+	/**
+	 * Clear query vars that cause issues with the main query - they are handled elsewhere.
+	 *
+	 * @since TBD
+	 *
+	 * @param WP $req The WP Object we're trying to prepare for.
+	 */
+	public function parse_request( $req ) {
+		if ( empty( $req->query_vars[ TEC::TAXONOMY ] ) ) {
```

Comment:
When we were in our call I forgot, we should add some checks here to make sure it only happens on our Views V2 request. Like not on the admin or if anyone else is using.

Reply:
✅